test/e2e/common/node/pod_resize.go

```diff
@@ -114,16 +114,19 @@ type patchSpec struct {
 	} `json:"spec"`
 }
 
-func supportsInPlacePodVerticalScaling(ctx context.Context, f *framework.Framework) bool {
-	node := getLocalNode(ctx, f)
+func isInPlacePodVerticalScalingSupportedByRuntime(ctx context.Context, c clientset.Interface) bool {
+	node, err := e2enode.GetRandomReadySchedulableNode(ctx, c)
+	framework.ExpectNoError(err)
 	re := regexp.MustCompile("containerd://(.*)")
```

Comment:
cri-o also supports IPPR at this point. What is the oldest version we run CI on? I think we should maybe just delete this.